### PR TITLE
avoid using using REDISMODULE_POSTPONED_ARRAY_LEN

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

**Comment by MeirShpilraien** [comment]
@mnunberg yes, I did this just to see that it fixes the network issue..

**Comment by filipecosta90** [comment]
hi there @MeirShpilraien and @MeirShpilraien . The change inplaced in this PR solved the network "issue". We have now a lot less packets to achieve the same query result. I've added the current master and the fix captures of 1 aggregate request. 

[1.6-fix_network_issue.pcapng.gz](https://github.com/RediSearch/RediSearch/files/3697140/1.6-fix_network_issue.pcapng.gz)
[1.6.3_master.pcapng.gz](https://github.com/RediSearch/RediSearch/files/3697141/1.6.3_master.pcapng.gz)

File: src/rlookup.c [hunk 1]
```diff
@@ -102,6 +102,25 @@ RLookupKey *RLookup_GetKey(RLookup *lookup, const char *name, int flags) {
   return RLookup_GetKeyEx(lookup, name, strlen(name), flags);
 }
 
+size_t RLookup_GetLength(RLookup *lookup, const RLookupRow *r, int explicitReturn) {
```
Comment (mnunberg) [inline src/rlookup.c]: - `const RLookup*`
- accept two arguments, `(uint32_t requiredFlags, uint32_t excludeFlags)`

Comment (mnunberg) [reply src/rlookup.c]:
```
if (requiredFlags && !(kk->flags & requiredFlags) { continue
if (excludeFlags && (kk->flags & excludeFlags) { continue
```

File: src/aggregate/aggregate_exec.c [hunk 2]
```diff
@@ -100,7 +100,9 @@ static size_t serializeResult(AREQ *req, RedisModuleCtx *outctx, const SearchRes
     count++;
     RLookup *lk = AGPLN_GetLookup(&req->ap, NULL, AGPLN_GETLOOKUP_LAST);
 
-    size_t nfields = RLookup_GetLength(lk, &r->rowdata, req->outFields.explicitReturn);
+    int excludeFlags =
```
Comment (mnunberg) [inline src/aggregate/aggregate_exec.c]: excludeFlags is F_HIDDEN, includeFlags is either 0 or F_EXPLICITRETURN

Comment (MeirShpilraien) [reply src/aggregate/aggregate_exec.c]: My bad

**Pull request merged**